codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/HttpBoundProtocolPayloadGenerator.kt

```diff
@@ -247,10 +247,11 @@ class HttpBoundProtocolPayloadGenerator(
         val fnName = "serialize_payload_${member.container.name.toSnakeCase()}"
         val ref = if (payloadMetadata.takesOwnership) "" else "&"
         val serializer = RuntimeType.forInlineFun(fnName, operationSerModule) {
-            val outputT = if (member.isStreaming(model)) "ByteStream" else "ByteSlab"
+            val outputT = if (member.isStreaming(model)) symbolProvider.toSymbol(member) else RuntimeType.ByteSlab.toSymbol()
```

Comment:
It looks like you can remove `"ByteSlab"` and `"ByteStream"` from the `codegenScope` map now since this was the only use.

Reply:
Will fix.